src/Dialogs/CaldavDialog.vala

```diff
@@ -397,7 +397,11 @@ public class OnlineAccounts.CaldavDialog : Hdy.Window {
             col.backend_name = "caldav";
 
             unowned var webdav = (E.SourceWebdav)source.get_extension (E.SOURCE_EXTENSION_WEBDAV_BACKEND);
+            #if HAS_EDS_3_45
```

Comment:
Do not indent the macro for better readability and consistency with the rest of our apps :slightly_smiling_face: